manifests/params.pp

```diff
@@ -141,7 +147,7 @@
       $libvirtd_srv = 'libvirtd'
       $libvirtd_cfg = '/etc/sysconfig/libvirtd'
       $libvirtd_source = 'puppet:///modules/one/libvirtd.sysconfig'
-      $rubygems       = ['builder']
+      $rubygems       = ['builder', 'sinatra']
```

Comment:
I found that the `sunstone` web UI refused to start, and threw a Ruby stack trace complaining about being unable to require `sinatra`  - perhaps a packaging bug in upstream's RPM's?